### ISSUE TITLE
Theses should not be requestable in the catalog

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangetheses#76]
Theses with a 'holding' appear as requestable in the catalog with a 'Reading room request' button. 

This is happening because based on the code there is a holding if there is a `pu.location` or a `dc.rights.accessRights` field in the record.

Please see the cases where a holding is hardcoded:
https://github.com/pulibrary/orangetheses/blob/4ac8dc2bd04b10db764fc37df3261531c9937061/lib/orangetheses/indexer.rb#L416
 
Also: 
https://github.com/pulibrary/orangetheses/blob/4ac8dc2bd04b10db764fc37df3261531c9937061/lib/orangetheses/indexer.rb#L294

Per @ccleeton 'there is no physical copy since 2013, anything [2012 and prior have a physical copy](https://catalog.princeton.edu/?utf8=%E2%9C%93&q=id%3Adsp*&search_field=all_fields&range%5Bpub_date_start_sort%5D%5Bbegin%5D=1968&range%5Bpub_date_start_sort%5D%5Bend%5D=2012)'

The code should be updated to satisfy that: There is no physical copy since 2013 anything 2012 and prior have a physical copy. Please ask @ccleeton for more info on this.

**Comment by ccleeton** [comment]
I see this is closed but I am not seeing it reflected in the catalog. Is there another ticket that this is still being worked on?
None of these should be requestable 
https://catalog.princeton.edu/?f%5Bformat%5D%5B%5D=Senior+thesis&f%5Blocation%5D%5B%5D=Mudd+Manuscript+Library&range%5Bpub_date_start_sort%5D%5Bbegin%5D=2013&range%5Bpub_date_start_sort%5D%5Bend%5D=2024&sort=pub_date_start_sort+asc%2C+title_sort+asc%2C+score+desc

**Comment by jrgriffiniii** [comment]
I apologize, I am looking to apply these updates into the production environment right at this moment.

**Comment by christinach** [comment]
@ccleeton we need [to resolve another ticket](https://github.com/pulibrary/orangetheses/issues/85) before we can update catalog production.

**Comment by christinach** [comment]
@LynnDurgin  @ccleeton 
Please review the  re-indexed theses on catalog-staging
[https://catalog-staging.princeton.edu theses with class year on 2012 and before](https://catalog-staging.princeton.edu/?q=id%3Adsp*&search_field=all_fields&range%5Bpub_date_start_sort%5D%5Bbegin%5D=1950&range%5Bpub_date_start_sort%5D%5Bend%5D=2012)

[https://catalog-staging.princeton.edu theses  class year 2013 and later](https://catalog-staging.princeton.edu/?q=id%3Adsp*&search_field=all_fields&range%5Bpub_date_start_sort%5D%5Bbegin%5D=2013&range%5Bpub_date_start_sort%5D%5Bend%5D=2024)

**Comment by ccleeton** [comment]
@LynnDurgin It looks like the embargoed theses for 2013-current are still acting as if there is a physical copy but unavailable. Is this ok?

**Comment by ccleeton** [comment]
We don't have any to my knowledge as the student choice to embargo only started in 2013. But lets wait for @LynnDurgin to chime in

**Comment by christinach** [comment]
@LynnDurgin @ccleeton I've reindexed theses with another change in catalog-qa. Please review both catalog-qa and [catalog-staging](https://github.com/pulibrary/orangetheses/issues/76#issuecomment-2412358236). Thank you.

[https://catalog-qa.princeton.edu 1924 to 2022](https://catalog-qa.princeton.edu/?q=id%3Adsp*&search_field=all_fields&range%5Bpub_date_start_sort%5D%5Bbegin%5D=1924&range%5Bpub_date_start_sort%5D%5Bend%5D=2012)

[https://catalog-qa.princeton.edu 2013 t0 2024](https://catalog-qa.princeton.edu/?q=id%3Adsp*&search_field=all_fields&range%5Bpub_date_start_sort%5D%5Bbegin%5D=2013&range%5Bpub_date_start_sort%5D%5Bend%5D=2024)

**Comment by LynnDurgin** [comment]
Hi @christinach , I have reviewed several theses on catalog-qa and all are behaving as discussed, i.e. all these from 2012 and earlier have holdings, all theses 2013 and later do not have holdings, regardless of access restriction. We discussed as potential future work to display embargo term (from pu.embargo.lift) or walkin access status (rights dc.rights.access.rights) on catalog record. Thank you!

**Comment by christinach** [comment]
Thanks @LynnDurgin ! I will proceed with what we've discussed and deploy this update to catalog production. 
I will also create a ticket for future work to add an access note for embargoes `that the item is embargoed until xxxx` as we've discussed on slack.

**Comment by christinach** [comment]
@ccleeton @LynnDurgin theses are now indexed in the production catalog. Thank you!

[Theses 1924 - 2012](https://catalog.princeton.edu/?q=id%3Adsp*&search_field=all_fields&sort=pub_date_start_sort+asc%2C+title_sort+asc%2C+score+desc&range%5Bpub_date_start_sort%5D%5Bbegin%5D=1924&range%5Bpub_date_start_sort%5D%5Bend%5D=2012)

[Theses 2013 - 2024](https://catalog.princeton.edu/?q=id%3Adsp*&search_field=all_fields&sort=pub_date_start_sort+asc%2C+title_sort+asc%2C+score+desc&range%5Bpub_date_start_sort%5D%5Bbegin%5D=2013&range%5Bpub_date_start_sort%5D%5Bend%5D=2024)
